content/tutorial-air-quality-analysis.md

```diff
@@ -0,0 +1,360 @@
+
+Air pollution is one of the most prominent types of pollution we face that has an immediate effect on our daily lives. The
+COVID-19 pandemic resulted in lockdowns in different parts of the world, offering a rare opportunity to study the effect of
+human activity (or lack there of) on air pollution. In this tutorial, we will study the air quality in Delhi, one of the
+worst affected cities by air pollution, before and during the lockdown from March to June 2020. For this, we will first compute
+the Air Quality Index for each hour from the collected pollutant measurements. Next, we will sample these indices and perform
+a [paired Student's t-test](https://en.wikipedia.org/wiki/Student%27s_t-test#Dependent_t-test_for_paired_samples) on them. It will statistically show us that the air quality improved due to the lockdown, supporting our intuition.
+
+Let's start by importing the necessary libraries into our environment. Also, enable IPython's [magic function](https://ipython.readthedocs.io/en/stable/interactive/magics.html#magic-matplotlib) to see the Matplotlib plots in the cell outputs.
+
+```{code-cell} ipython3
+import numpy as np
+from scipy import stats
+import matplotlib.pyplot as plt
+
+%matplotlib inline
+```
+
+## Building the dataset
+
+We will use a condensed version of the [Air Quality Data in India](https://www.kaggle.com/rohanrao/air-quality-data-in-india) dataset. This dataset contains air quality data and AQI (Air Quality Index) at hourly and daily level of various stations across multiple cities in India. The condensed version available with this tutorial contains hourly pollutant measurements for Delhi
+from May 31, 2019 to June 30, 2020. It has measurements of the standard pollutants that are required for Air Quality Index calculation and a few other important ones:
+Particulate Matter (PM 2.5 and PM 10), nitrogen dioxide (NO2), ammonia (NH3), sulfur dioxide (SO2), carbon monoxide (CO), ozone (O3), oxides of nitrogen (NOx), nitric oxide (NO), benzene, toluene, and xylene.
+
+Let's print out the first few rows to have a glimpse of our dataset.
+
+```{code-cell} ipython3
+! head air-quality-data.csv
+```
+
+For the purpose of this tutorial, we are only concerned with standard pollutants required for calculating the AQI, viz., PM 2.5, PM 10, NO2, NH3, SO2, CO, and O3. So, we will only import these particular columns with [np.genfromtxt](https://numpy.org/devdocs/reference/generated/numpy.genfromtxt.html). We'll initially create two sets: `pollutants_A` with PM 2.5, PM 10, NO2, NH3, and SO2, and `pollutants_B` with CO and O3. The
+two sets will be processed slightly differently, as we'll see later on.
+
+```{code-cell} ipython3
+pollutants_A = np.genfromtxt("air-quality-data.csv", dtype='f8', delimiter=",",
+                             skip_header=1, usecols=(1, 2, 3, 4, 5))
+pollutants_B = np.genfromtxt("air-quality-data.csv", dtype='f8', delimiter=",",
+                             skip_header=1, usecols=(6, 7))
```

Comment:
Just a nit - it'd be more efficient to only read from file once and then slice out the columns you'd like to use, e.g. something like:

```suggestion
pollutant_data = np.loadtxt("air-quality-data.csv", dtype=float, delimiter=",",
                            skiprows=1, usecols=range(1, 8))
pollutants_A = pollutant_data[:, 0:5]  # PM2.5, PM10, NO2, NH3, and SO2
pollutants_B = pollutant_data[:, 5:]  # CO and O3
```

Note also the use of `loadtxt` instead of `genfromtext`, which will (hopefully) be much faster in the next NumPy release!